src/_components/shortcodes/banner/banner.js

```diff
@@ -0,0 +1,97 @@
+const svgButton = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="72" height="72" viewBox="0 0 72 72" aria-hidden="true">
+<defs>
+  <clipPath id="clip-Button_play">
+    <rect width="72" height="72"/>
+  </clipPath>
+</defs>
+<g id="Button_play" clip-path="url(#clip-Button_play)">
+  <rect width="72" height="72" fill="transparent"/>
+  <g id="Group_503" data-name="Group 503" transform="translate(-0.204)">
+    <circle id="Ellipse_4" data-name="Ellipse 4" cx="36" cy="36" r="36" transform="translate(0.204)" fill="#fe796a"/>
+    <path id="Path_120" data-name="Path 120" d="M35.213,0V13.347L17.606,25.291,0,13.347V0Z" transform="translate(26.874 53.282) rotate(-90)"/>
+  </g>
+</g>
+</svg>
+`;
+const style = `
```

Comment:
Is it possible to store the css in a .css file?

Reply:
I have been checking that as well; I have put CSS in the file, but it did not attach to the project by using short-code in pages; I checked the configuration and spent some time to find a way to change the config, but I did not find a way to solve that. You can check other shortcodes as well. There are no styles attached to the main CSS after serving.
I even put the same CSS in the file next to the component in my PR.
I would say it stems from a lack of configuration for attaching CSS.